requirements.txt

```diff
@@ -1,21 +1,52 @@
+argcomplete==2.0.0
```

Comment:
Are all these new dependencies caused by `inflector` ?

Reply:
I honnestly don't know. But i don't think so. It should be the dependencies of packages I tested in during development, like `django-jsonform` and others...

I executed `pip uninstall <package>` but it doesn't seem to remove dependencies...

Reply:
I made the changes